Skps sdk json6

shriku11 opened the pull request:
<!-- DO NOT DELETE THIS TEMPLATE -->

## Description
<!--
Changed SDK to include new Action group actions. Also added changes for patch operation,
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/2077
-->

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

Commit:
Added SDK changes for ActionGroup Changes

Commit:
added change log entry

azuresdkci commented:
Can one of the admins verify this patch?

azuresdkci commented:
Can one of the admins verify this patch?

azuresdkci commented:
Can one of the admins verify this patch?

azuresdkci commented:
Can one of the admins verify this patch?

Commit:
fixed the Nuget.Config

shahabhijeet commented:
@shriku11 can you onboard to azure GitHub using aka.ms/azuregithub
Also please provide link to the spec and also can you fix the title of the PR so that we know what this change is all about.
As soon as you onboard, you will be authenticated to run CI.

shriku11 commented:
@shahabhijeet I am already onboarded to GitHub using aka.ms/azuregithub
If you click on edit for 1st comment by me in PR that is template I filled you will see in description section what this PR is about and link to Swagger doc. I am also adding it here,

## Description
<!--
Changed SDK to include new Action group actions. Also added changes for patch operation,
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/2077
-->

shriku11 commented:
Could someone start reviewing this PR? Thanks!

shahabhijeet commented:
@azuresdkci add to whitelist

shriku11 commented:
Joined the Azure org, how do I trigger the CI now? I have no new changes to push. As far as I know this only triggers if I add new changes.

shahabhijeet commented:
@azuresdkci test this please

Commit:
Commented the unecessary line

shahabhijeet reviewed (changes requested):
Monitor Tests are failing.
Search for Monitor.test.trx file and open it in VS to see failures.
https://azuresdkci.westus2.cloudapp.azure.com/job/NetCore-SdkCI/1128/artifact/TestResults/netCore11/

Commit:
Skipped unecessary test

Commit:
Revrted the version change for Newtonsoft library.

shriku11 commented:
@shahabhijeet , any idea how long this CI should take? This seems stuck for last 4 hours.

shriku11 commented:
@shahabhijeet , The tests are fixed. Could you take a look and if good merge this PR?

shahabhijeet commented on src/SDKs/Monitor/Management.Monitor/Properties/AssemblyInfo.cs:
```diff
@@ -3,16 +3,15 @@
 
-[assembly: AssemblyTitle("Microsoft Azure Monitor Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure Monitor operations.")]
+// [assembly: AssemblyTitle("Microsoft Azure Monitor Library")]
```
@shriku11 please uncomment these lines.
Also any reason you are commenting assembly meta data.

shriku11 replied:
This is fixed and added the commented line back. The local build was giving error and was suggesting to remove this.

shahabhijeet commented on src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj:
```diff
@@ -22,7 +22,7 @@
 	<ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
 		<Reference Include="System.Net.Http" />
-		<PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
+    <PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
```
@shriku11 if the only change you made is to delete indentation, can you undo this, so that there is no history of this change plus the indentation is back to normal.

shriku11 replied:
Fixed the indentation.

Commit:
Fixed a comment from SDK team

shriku11 committed:
Update Monitor.Tests.csproj

Commit:
Changed the Assemblyinfo.cs to change the version.
